frontend/tailwind.config.ts

```diff
@@ -226,7 +226,6 @@ export default {
       },
       borderRadius: {
         inherit: "inherit",
-        md: "0.5rem",
```

Comment:
I didn't realize that `rounded-lg` is actually `border-radius: 0.5rem` when I added this new value, `md`. Since `md` is redundant now, I removed it and replaced it in the components with `rounded-lg`